akka-docs-dev/rst/scala/code/docs/http/scaladsl/HttpServerExampleSpec.scala

```diff
@@ -0,0 +1,256 @@
+/*
+ * Copyright (C) 2009-2014 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package docs.http.scaladsl
+
+import scala.concurrent.Future
+import org.scalatest.{ WordSpec, Matchers }
+import akka.actor.ActorSystem
+
+class HttpServerExampleSpec extends WordSpec with Matchers {
+
+  "binding-example" in {
```

Comment:
Nope, not needed. We are using our own `includecode` sphinx extension for the HTTP docs, which is a bit smarter that the "traditional" Akka one... :)